examples/azure-rules.yaml

```diff
@@ -656,4 +656,21 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
+        scored: true
+  - id: 1.6
+    description: "Check OS and package versions are supported and up-to-date"
```

Comment:
well this is only checking "supported" not "up to date"

Reply:
It's a description for the id 1.6, the 1.6.1 checks only the supported os release, there will come 1.6.2 and 1.6.3 in #31  that check that the packages are up-to-date.

Reply:
ok got it. I was already confused why these are two PRs rather than one iwth multiple commits. fine by me.